src/lib/accessibility/Behaviors/Chat/chatBehavior.ts

```diff
@@ -36,8 +36,11 @@ const ChatBehavior: Accessibility = (props: any) => ({
 })
 
 const getLastTabbableElement = (root: HTMLElement): HTMLElement => {
+  const lastVisibleMessage = root.querySelector('[data-last-visible]') as HTMLElement
```

Comment:
note: this ```chatBehavior``` would be moved from this repo in the future

Reply:
Could this be changed to
```suggestion
  const lastVisibleMessage = root.querySelector('[data-last-visible="true"]') as HTMLElement
```
When selecting for `[data-last-visible]` it would select elements that have both `data-last-visible="true"` and `data-last-visible="false"` set.

It is also hard to just render the attribute without values in react as I would need to create a new fork for the whole component. e.g.

```
if(lastVisible) {
    return <Chat data-last-visible />
} else {
    return <Chat />
}
```